src/main/java/com/github/tomakehurst/wiremock/common/url/PathTemplate.java

```diff
@@ -28,7 +28,7 @@
 
 public class PathTemplate {
   static final Pattern SPECIAL_SYMBOL_REGEX =
-      Pattern.compile("(?:\\{(?<variable>[^}]+)\\})|(?<wildcard>\\*\\*)");
+      Pattern.compile("\\{(?<variable>[^}]+)}|(?<wildcard>\\*\\*)");
```

Comment:
IntelliJ told me these regex syntaxes were unnecessary, and removing them left all the tests passing, so I believed it.